content/en/synthetics/settings/_index.md

```diff
@@ -68,19 +70,40 @@ Note: Variable values are updated whenever the test they are extracted from runs
 
 {{< /tabs >}}
 
+### Permissions
+
+By default, only users with the [Datadog Admin and Datadog Standard roles][3] can access the Synthetic Monitoring **Global Variables** page. You can get access to the **Global Variables** page by having your user upgraded to one of these two [default roles][link]. 
```

Comment:
Looks like we're missing a link in there as well for default roles

Reply:
```suggestion
By default, only users with the [Datadog Admin and Datadog Standard roles][3] can access the Synthetic Monitoring **Global Variables** page. You can get access to the **Global Variables** page by having your user upgraded to one of these two [default roles][13]. 
```